uint32 max_box
---
geometry_msgs/PoseArray boxes
